app/src/main/java/org/permanent/permanent/ui/archiveOnboarding/compose/ArchiveOnboardingScreen.kt

```diff
@@ -36,69 +40,75 @@ fun ArchiveOnboardingScreen(
     viewModel: ArchiveOnboardingViewModel
 ) {
     val context = LocalContext.current
-    val horizontalPaddingDp = 32.dp
-    val spacerPaddingDp = 8.dp
     val blue900Color = Color(ContextCompat.getColor(context, R.color.blue900))
     val blueLighterColor = Color(ContextCompat.getColor(context, R.color.blueLighter))
     val pagerState = rememberPagerState(initialPage = 0)
+    val windowWidthSizeClass by remember { mutableStateOf(viewModel.getWindowWidthSizeClass()) }
+
+    val horizontalPaddingDp =
```

Comment:
You could just store a Bool value in the viewModel like val isExpanded or isTablet and use that.